SPECS/mariner-rpm-macros/macros.dist

```diff
@@ -0,0 +1 @@
+%dist @DIST@
```

Comment:
I see the PR check is now failing. I think we might just need a small tweak inside `rpmops.sh` where we replace

```bash
rpmspec "${DEFINES[@]}" -D "_sourcedir $sourcedir" "${MACROS[@]}" "$@"
```

with

```bash
rpmspec "${MACROS[@]}" "${DEFINES[@]}" -D "_sourcedir $sourcedir" "$@"
```